### PR TITLE
Added Russian translation

### PR DESCRIPTION
**Pull request by RusFjord**

### CODE REVIEW

File: src/app/engines/i18n.engine.ts [hunk 1]
```diff
@@ -70,6 +72,7 @@ class I18nEngine {
         i18next.addResources('sk-SK', 'translation', TRANSLATION_SK_SK);
         i18next.addResources('zh-CN', 'translation', TRANSLATION_ZH_CN);
         i18next.addResources('zh-TW', 'translation', TRANSLATION_ZH_TW);
+        i18next.addResources('zh-TW', 'translation', TRANSLATION_RU_RU);
```
Comment (vogloblinsky) [inline src/app/engines/i18n.engine.ts]: Could you fix this line please ;)

Comment (RusFjord) [reply src/app/engines/i18n.engine.ts]: Thanks. I changed key name

Comment (vogloblinsky) [reply src/app/engines/i18n.engine.ts]: Thanks

**Comment by vogloblinsky** [comment]
Can you add one test in this file please ?
```https://github.com/compodoc/compodoc/blob/develop/test/src/cli/cli-i18n.spec.ts```

**Comment by RusFjord** [comment]
> Can you add one test in this file please ? `https://github.com/compodoc/compodoc/blob/develop/test/src/cli/cli-i18n.spec.ts`

Yes. I added

**Comment by vogloblinsky** [comment]
Thanks for the PR ! 🙏🏻

**Pull request merged**